### PR TITLE
fix: Fix comment about semantic-release

### LINKED ISSUE #302: fix comment about semantic release
https://github.com/osl-incubator/scicookie/blob/main/src/scicookie/%7B%7Bcookiecutter.project_slug%7D%7D/build-system/poetry/pyproject.toml#L9

it should be:

```diff
- version = "{{ cookiecutter.project_version }}" #semantic-release
+ version = "{{ cookiecutter.project_version }}"  # semantic-release
```

### PR DESCRIPTION
**Pull request by YurelyCamacho**
## Pull Request description
Fixing comment about semantic-release in poetry/pyproject.toml file

## Pull Request checklists

This PR is a:

- [X] bug-fix
- [ ] new feature
- [ ] maintenance

About this PR:

- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:

- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more
      complexity.
- [ ] New and old tests passed locally.

## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved
```

### CODE REVIEW

**Comment by YurelyCamacho** [comment]
@xmnlab When running the test that fails locally it appears:
```
16:00 $ ./tests/smoke/linters.sh
-------------------- Smoke test for use_bandit=yes -------------------
+ PATH_ORI=/home/usuario/miniconda3/envs/scicookie/bin:/home/usuario/miniconda3/condabin:/home/usuario/.sdkman/candidates/sbt/current/bin:/home/usuario/.sdkman/candidates/java/current/bin:/home/usuario/.pyenv/plugins/pyenv-virtualenv/shims:/home/usuario/.pyenv/shims:/home/usuario/.pyenv/bin:/home/usuario/miniconda3/envs/tesis/bin:/home/usuario/miniconda3/condabin:/home/usuario/.poetry/bin:/home/usuario/.local/bin:/usr/local/bin:/usr/bin:/var/lib/snapd/snap/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/usuario/.local/share/coursier/bin
++ pwd
+ PWD_ORI=/home/usuario/projects/cookiecutter-python
+++ dirname /home/usuario/projects/cookiecutter-python/tests/smoke/base.sh
++ cd /home/usuario/projects/cookiecutter-python/tests/smoke
++ cd ../..
++ pwd
+ PROJECT_PATH=/home/usuario/projects/cookiecutter-python
++ ensureconda --conda --no-install
++ sed 's:/mamba$:/conda:'
/home/usuario/projects/cookiecutter-python/tests/smoke/base.sh: línea 9: ensureconda: orden no encontrada
+ CONDA_PATH=
++ ensureconda --mamba --no-install
++ sed 's:/conda$:/mamba:'
/home/usuario/projects/cookiecutter-python/tests/smoke/base.sh: línea 10: ensureconda: orden no encontrada
+ MAMBA_PATH=
+ [[ ! -f '' ]]
+ echo '[EE] '\''mamba'\'' not found.'
[EE] 'mamba' not found.
+ exit 1
```

**Comment by xmnlab** [comment]
Could you also change the pre-commit-config.yaml in the template side, chaging `ruff` linter to `ruff check`?


About the error you mentioned. Do you have mamba locally?

**Comment by xmnlab** [comment]
> Could you also change the pre-commit-config.yaml in the template side, chaging `ruff` linter to `ruff check`?
> 
> About the error you mentioned. Do you have mamba locally?

a friendly reminder about this

**Comment by YurelyCamacho** [comment]
@xmnlab Sorry I had not seen this until now. I'm going to do it

**Comment by YurelyCamacho** [comment]
@xmnlab Done

File: src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml [hunk 1]
```diff
@@ -46,9 +46,9 @@ repos:
           - python
 {% endif %}
 {%- if cookiecutter.use_ruff_linter == 'yes' %}
-      - id: ruff-linter
-        name: ruff-linter
-        entry: ruff check --fix
+      - id: ruff check
```
Comment (xmnlab) [inline src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml:49]: @YurelyCamacho , could you move back to the original version of this file please?
this was already fixed on the main branch.
thanks!

Comment (YurelyCamacho) [reply src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml]: @xmnlab Done

**Comment by xmnlab** [comment]
Thanks @YurelyCamacho 
Just to give some context, semantic relase   is configured to search by a specific string in that file and uses "  # semantic-release" as part of its search .. anything different would return nothing. That it is why it is important, it is not just a comment :)

**Pull request merged**